### ISSUE TITLE
Usage Question: How to avoid stray pixels from  tft.drawString()

### ISSUE THREAD

**Issue by Gerry33** [Bodmer/TFT_eSPI#60]
How  can I avoid such stray pixels from tft.drawString(...);

![straypixel](https://user-images.githubusercontent.com/686034/33428532-87b06e8a-d5c9-11e7-9850-4c79b4e6c4c4.jpg)

Text datum is set to  ` 'setTextDatum (TL_DATUM)',`   
    `tft.setTextPadding(tft.textWidth(("00:00"))` is always set to the maximum widh possible.

**Comment by Bodmer** [comment]
Hi, can you provide a more complete code example that illustrates the problem, including the font used. Thanks.

**Comment by Gerry33** [comment]
Hi Bodmer,
thank you for taking care.

Here is a sketch that demonstrates:
```
#include <SPI.h>
#include <TFT_eSPI.h>

TFT_eSPI tft = TFT_eSPI();
void setup()
{
	Serial.begin(115200);
	delay (10);
	Serial.printf("\nStarted:" __FILE__ ", ESP-Core:<%s>,\n ESP-SDK:<%s>\n", ESP.getCoreVersion().c_str(), ESP.getSdkVersion());
	tft.begin();
	tft.setRotation(2);
	tft.fillScreen		(TFT_BLACK);
	tft.setTextColor	(TFT_YELLOW, TFT_BLACK);
	tft.setFreeFont		(&FreeSerifBold24pt7b); // this font comes with the lib
	tft.setTextDatum	(TL_DATUM);
	tft.setTextPadding(tft.textWidth("00:00"));
}
void loop()
{
		tft.drawString("00:00", 10, 10);
		delay (3000);
		tft.drawString("12:34", 10, 10);	// stray pixels
		delay (3000);
}
```
The '12:34' String shows like this
![img_20171130_181852](https://user-images.githubusercontent.com/686034/33444866-aba23b3e-d5fb-11e7-9f24-dc3506993562.jpg)

The left stray pixels are remainers of the '00:00' - string.

**Comment by Bodmer** [comment]
I have fixed this is the latest version 0.17.15 in the repository.

Thanks for pointing out this issue.

**Comment by Gerry33** [comment]
Perfect ! 

> Thanks for pointing out this issue

I'm the one to give a big thank to you !   

Gerry
